Core.DomainModel/ItContract/ItContract.cs

```diff
@@ -17,8 +16,11 @@ namespace Core.DomainModel.ItContract
     /// <summary>
     ///     Contains info about an it contract
     /// </summary>
-    public class ItContract : HasRightsEntity<ItContract, ItContractRight, ItContractRole>, IHasReferences, IHierarchy<ItContract>, IContractModule, IOwnedByOrganization, IHasName, IEntityWithExternalReferences, IEntityWithAdvices, IEntityWithUserNotification
+    public class ItContract : HasRightsEntity<ItContract, ItContractRight, ItContractRole>, IHasReferences, IHierarchy<ItContract>, IContractModule, IOwnedByOrganization, IHasName, IEntityWithExternalReferences, IEntityWithAdvices, IEntityWithUserNotification, IHasUuid
     {
+        public const string InOperationAgreementElementName = "Drift";
```

Comment:
Det kan vi ikke leve med ;-(
Det lader til at vi er nødt til at udstille aftaleelementerne (udfaldsrum på kontrakten) for den der kan gå i stykker